site-scrapers/Atrius.js

```diff
@@ -12,7 +13,29 @@ module.exports = async function GetAvailableAppointments() {
     };
 };
 
+function urlRedirect(url, options) {
+    return new Promise((resolve) => {
+        let response = "";
+        https.get(url, options, (res) => {
+            let body = "";
+            res.on("data", (chunk) => (body += chunk));
+            res.on("end", () => {
+                response = res.headers?.location;
+                resolve(response);
+            });
+        });
+    });
+}
+
 async function ScrapeWebsiteData() {
+    const checkSlots = await urlRedirect(sites.Atrius.website, {});
+    if (checkSlots.match("No_Slots")) {
+        console.log(
+            `Atrius redirecting to no slots, ${checkSlots}, assuming failure!`
+        );
+        return false;
```

Comment:
we don't want to return false , I believe we want 
```
{
    hasAvailability: false,
    availability: {} //this line is optional
}
```

Reply:
Done.
(What goes wrong if we don't have  the `hasAvailability` key?)

Reply:
possible it would work correctly but I'm in the process of making a test that checks the shape of all returned values from main.js so that'll help catch things like this.